clients/samples/hipfft/hipfft_planmany_2d_r2c.cpp

```diff
@@ -1,103 +1,67 @@
-#include <iomanip>
-#include <iostream>
-
-#include <fftw3.h>
+#include <complex>
 #include <hip/hip_runtime.h>
 #include <hipfft.h>
+#include <iostream>
+#include <vector>
 
 int main()
 {
-    ///------------------------------------------------------------------------
-    /// fftwf_plan_many_dft_r2c vs hipfftPlanMany
-    /// 2D R2C forward in-place
+    int rank    = 2;
+    int n[2]    = {4, 5};
+    int howmany = 3; // batch size
```

Comment:
It is better to leave the comment "forward in-place"

Reply:
Commit 6e28ab0 adds console output which also serves as a comment (also for z2z case).